dynamiqs/solvers/ode/adaptive_solver.py

```diff
@@ -205,9 +205,12 @@ def integrate_augmented(
             y, a = new_leaf_tensor(y), new_leaf_tensor(a)
```

Comment:
Pretty sure we can add back the following now (it's really this line that enforces the graph to be disconnected between step n and n+1)
```suggestion
            # the computation graph attached to `y`, `a`, `fyt` and `fat` is
            # automatically freed after next line, because there are no more references
            # to the original tensors (the old `y`, `a`, `fyt` and `fat` go out of
            # scope)
            y, a, fyt, fat = (
                new_leaf_tensor(y),
                new_leaf_tensor(a),
                new_leaf_tensor(fyt),
                new_leaf_tensor(fat),
            )
```

Reply:
I mean, we can add it, but it's useless I think. Here you are intantiating `fyt` as a leaf tensor, and in the next line, we're saying that `fyt` is instead `self.odefun_backward(t, y)`. So we're assigning another value.

Reply:
Correct!